### DISCUSSION TITLE
error 400

### DISCUSSION THREAD

**Discussion by muslmasan** [serverpod/serverpod#1289] [Q&A]
**Describe the bug*
when i run a mobile application in the physical device i get error
I/flutter (18392): ServerpodClientException: , statusCode = 400

**To Reproduce**
1. add my computer ip address to teh code
2. Click on submet
3. i get error 400 

**Serverpod versions**
 - 1.1.1
 - _Library version: 1.1.1

**Platform information**
windows / android

**Comment by Isakdl** [comment]
Can you please add some code example and a description of what you are trying to do.

**Comment by muslmasan** [comment]
ok i just try to run serverpod in physical device not in emulator and the phone is connected with the computer with wifi and usb

**Comment by muslmasan** [comment]
I/flutter (18392): ServerpodClientException: Internal server error. Call log id: 2, statusCode = 500

**Comment by Isakdl** [comment]
> I/flutter (18392): ServerpodClientException: Internal server error. Call log id: 2, statusCode = 500

Suggests that something is crashing in your server. Hard to tell without seeing your implementation.
